### PR TITLE
feat: add `bohemia.studio`

### PR DESCRIPTION
**Pull request by prontozz**
- [x] add to population script 0475075f3d69c9dc4837eeb8b937a11eba8cd5cd, cfd386c5286af6b58025f6daee87f51aededc4b8
- [x] add to tests  e636da0dc55c202295d695a08c0e4365b5a808e5

### CODE REVIEW

File: tests/postgres.test.js [hunk 1]
```diff
@@ -242,19 +243,39 @@ describe(config.host, () => {
...
+          `INSERT INTO bohemia.studio("studioName", description, platform, "studioHead", "addressFK", phone, "alternatePhone", email) VALUES ('Big Zebra Solutions', 'Recently secured funding from Sony to create a AAA 3rd person adventure game. Their previous titles were minimalist art-as-games experiences.', 'Console', 'Maggie Shaw', 5, 1483960457, 07700900667, 'maggie.shaw@bigzebrasolutions.com')`,
+          `INSERT INTO bohemia.studio("studioName", description, platform, "studioHead", "addressFK", phone, "alternatePhone", email) VALUES ('Unhinged', 'A small team who create games using assets from 3D model repositories. Unhinged concentrate mainly on games that make use of their in-house Eagle Engine.', 'PC', 'Yasmin Singh', 6, 1614960928, 07700900299, 'yasmin.singh@unhinged.co.uk')`
+        ].join(';')
+      );
+
+      const res = await client.query('SELECT * FROM bohemia.studio');
+
+      expect(res.command).toBe('SELECT');
+      expect(res.rowCount).toBe(6);
```
Comment (c0reme) [inline tests/postgres.test.js:278]: More tests on the columns are needed here, something on the lines of
```js
res.rows.forEach((row) => {
  expect(row).toHaveProperty('studioName');
  // ....
});
```

**Review by c0reme — CHANGES_REQUESTED** [review]
Not bad, please could you make sure these are changed then request a review afterwards.

File: scripts/CreationScript.sql [hunk 2]
```diff
@@ -43,12 +43,13 @@ create table if not exists bohemia.studio
...
+    "studioHead" varchar(255) not null,
     "addressFK" bigint not null constraint studio_address_addressId_fk references bohemia.address,
-    "studioHead" bigint not null constraint studio_employee_employeeId_fk references bohemia.employee,
-    phone int not null,
-    "alternativePhone" int null,
-    email varchar(255) not null,
-    platform varchar(255) not null
+    phone bigint not null,
+    "alternativePhone" bigint null,
+    email varchar(255) not null
```
[the remark this answers is not in the document]
Comment (c0reme) [reply scripts/CreationScript.sql:52]: Updated locally with 8fa40113ebd7202f7a53a2d39bedcc7c608357f0

File: tests/postgres.test.js (hunk 1 above)
Comment (prontozz) [reply tests/postgres.test.js:278]: Something along the lines of this?

`res.rows.forEach((row) => {
        expect(row).toHaveProperty('studioName');
        expect(row).toHaveProperty('description');
        expect(row).toHaveProperty('studioHead');
        expect(row).toHaveProperty('addressFK');
        expect(row).toHaveProperty('phone');
        expect(row).toHaveProperty('alternatePhone');
        expect(row).toHaveProperty('email');
      });`

**Pull request merged**